docs/cli.md

```diff
@@ -38,6 +38,7 @@ The usage of the `postgraphql` binary is as follows. To pull up this documentati
     -a, --classic-ids                use classic global id field name. required to support Relay 1
     -j, --dynamic-json               enable dynamic JSON in GraphQL inputs and outputs. uses stringified JSON by default
     -M, --disable-default-mutations  disable default mutations, mutation will only be possible through Postgres functions
+    -l, --size-limit                 set the maximum size of JSON bodies that can be parsed (default 100kB)
```

Comment:
Could we change the name to `--body-size-limit`? Could we also add a note that you can define it using a pretty size like `200kB`.